patches/chromium/picture-in-picture.patch

```diff
@@ -9,23 +9,32 @@ don't get errors for Chrome's generated resources, which are non-existent
 because we don't generate them in our build.
 
 diff --git a/chrome/browser/ui/views/overlay/close_image_button.cc b/chrome/browser/ui/views/overlay/close_image_button.cc
-index 91ef40b1c054ffceec006400c18d68b5ab393161..7a33b6e01b57b58a77ec54be59f6f3235dab936c 100644
+index 8168b4cfbafd42fa93a5aa9a3691c2552fabfb86..2c3d2d63f0a7307bf28aa3d630aa9c3fb9931de1 100644
 --- a/chrome/browser/ui/views/overlay/close_image_button.cc
 +++ b/chrome/browser/ui/views/overlay/close_image_button.cc
-@@ -4,9 +4,12 @@
- 
+@@ -5,9 +5,9 @@
  #include "chrome/browser/ui/views/overlay/close_image_button.h"
  
+ #include "base/feature_list.h"
 +#include "build/branding_buildflags.h"
  #include "chrome/browser/ui/color/chrome_color_id.h"
  #include "chrome/grit/generated_resources.h"
-+#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
- #include "components/vector_icons/vector_icons.h"
-+#endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING)
+-#include "components/vector_icons/vector_icons.h"
+ #include "media/base/media_switches.h"
  #include "ui/base/l10n/l10n_util.h"
  #include "ui/base/metadata/metadata_impl_macros.h"
- #include "ui/base/models/image_model.h"
-@@ -26,7 +29,10 @@ CloseImageButton::CloseImageButton(PressedCallback callback)
+@@ -16,6 +16,10 @@
+ #include "ui/views/accessibility/view_accessibility.h"
+ #include "ui/views/vector_icons.h"
+ 
++#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
++#include "components/vector_icons/vector_icons.h"
++#endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING)
++
+ namespace {
+ 
+ constexpr int kCloseButtonTopMargin = 5;
+@@ -29,7 +33,10 @@ CloseImageButton::CloseImageButton(PressedCallback callback)
```

Comment:
Looking at this patch update, the 
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#include "components/vector_icons/vector_icons.h"
+#endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING)
+
should have been done inline like the patch previously did instead of deleting the line and adding it back further down in the file.

Reply:
This change was made in an attempt to possibly reduce conflicts in the future. I think if the diff has one line being deleted up top, git might be more able to resolve without conflict?

I can revert this if it's not desirable. Lmk!

Reply:
Generally speaking we shouldn’t be moving stuff unnecessarily around in the files we are patching because it can create confusion when someone else has to update the patch.  The conflict could happen if either the lines around the deleted line OR the added lines are modified so placement doesn’t really help with conflicts.

Reply:
Agreed, it's likely the changed I made might not actually help. Reverted in [e15756a](https://github.com/electron/electron/pull/44748/commits/e15756ab949763904d02b4023f4242c8c3612924)